spec/users_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
```

Comment:
- [ ] You should have `users_spec.rb` in `spec/requests/` directory with `type: request`.